sdk/typescript/test/connect.spec.ts

```diff
@@ -149,7 +149,7 @@ describe("TypeScript sdk Connect", function () {
     }
   })
 
-  describe.skip("Automatic Provisioned CLI Binary", function () {
+  describe("Automatic Provisioned CLI Binary", function () {
```

Comment:
Yeah, this was because of the dev module, but it's required by mage still. Better that dev needs fixing instead of mage.